_pages/en/strategy-build-vs-buy.md

```diff
@@ -132,31 +149,48 @@ Stakeholders are expected to still comply with existing policy instruments inclu
+    - Contract obligations with vendors
+    - Intellectual property limitations
+    - Proprietary and closed formats
+  - Decreased ratio of IT Solutions and IT Products for which the department can not choose a service integrator and/or a software maintainer other than the group that provides the software licence.
+  - Decreased use of IT Solutions and IT Products for which there are non-interoperable and exclusive:
+    - File formats
+    - Application Programming Interfaces
+    - Data exports
+- Building
+  - Increased ratio of internally built projects released as OSS
 - Applicable to all scenarios
-  - Increased speed and flexibility for changes
+  - Increased ratio of decoupled IT Solutions and IT Products compared to baseline
+  - Increased number of APIs accessible for reuse throughout ESDC IT Solutions and IT Products
+  - Increased number of IT Solutions and Products migrated to and operated on cloud environments (Cloud First Strategy)
+  - Increased speed and flexibility for software changes deployed to productions for IT Solutions and IT Products (see Target Solution Delivery Model)
 
 ## Appendix A - Business Case (Diagnostic)
 
 ### Build vs Buy : Oversimplified Question<!-- omit in toc -->
 
 The creation of this strategy document stems from a request to help IITB decide whether it should prioritize buying an IT Solution over building one.
 
-Looking for a default answer and approach to the question "Should an IT Solution be built or bought?" at the organizational level is a mental shortcut that bypasses GoC Mandatory Procedures for [Enterprise Architecture Assessments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) and [on Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604) as well as the concepts of solution architecture, where the relevant experts of the organization, both from business lines, finance, security, information technology, etc., work together to design a solution while leveraging various resources, including people, tools, processes, and rules.
+Looking for a default answer and approach to the question "Should an IT Solution be built or bought?" at the organizational level is a mental shortcut that bypasses GoC Mandatory Procedures for [Enterprise Architecture Assessments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) and on [Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604) as well as the concepts of solution architecture, where the relevant experts of the organization, both from business lines, finance, security, information technology, etc., work together to design a solution while leveraging various resources, including people, tools, processes, and rules.
 
 An IT solution is by default built, whether by a single person, multiple teams or by a third party, and it will most often consist in an assemblage of many components, known as IT Products in this document.
 
```

Comment:
Will need to update the diagram to reflect Guiding Policy's Governance, Compliance, and Reporting #1. ("IT Solutions are broken down into manageable parts (IT Products) ")

So change "IT Component" to "IT Product".

Reply:
Need to update the image, I think you had the original @remyb2canada ?